### ISSUE TITLE
Annotation missing from AmiGO - and not in the reports

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-site#1105]
Hello, 

Reported by @RLovering 

This annotation is in QuickGO but not in AmiGO. It's not in the error reports (at least not in http://release.geneontology.org/2019-06-09/reports/goa_human-report.html)

UniProtKB	P05067	APP		GO:0070206	PMID:19660551	IMP		P	Amyloid-beta precursor protein	APP,A4,AD1	protein	taxon:9606	20170710	ARUK-UCL	occurs_in(UBERON:0000955),part_of(GO:0007568),has_input(UniProtKB:P05067:PRO_0000000093),has_output(ComplexPortal:CPX-1120),has_output(ComplexPortal:CPX-1062)	UniProtKB:P05067:PRO_0000000092

Can someone check what's happening ? 

Thanks, Pascale

**Comment by kltm** [comment]
Most basically:
```
bbop@wok:/home/skyhook/release/products/annotations$ reset && zgrep "20170710" *-src.gaf.gz | grep "GO:0070206"
goa_uniprot_all-src.gaf.gz:UniProtKB	P05067:PRO_0000000092	APP		GO:0070206	PMID:19660551	IMP		P	Amyloid-beta precursor protein	APP|A4|AD1	protein	taxon:9606	20170710	ARUK-UCL	occurs_in(UBERON:0000955),part_of(GO:0007568),has_input(UniProtKB:P05067:PRO_0000000093),has_output(ComplexPortal:CPX-1120),has_output(ComplexPortal:CPX-1062)	UniProtKB:P05067:PRO_0000000092
goa_uniprot_all-src.gaf.gz:UniProtKB	P05067:PRO_0000000093	APP		GO:0070206	PMID:19660551	IMP		P	Amyloid-beta precursor protein	APP|A4|AD1	protein	taxon:9606	20170710	ARUK-UCL	occurs_in(UBERON:0000955),part_of(GO:0007568),has_input(UniProtKB:P05067:PRO_0000000092),has_output(ComplexPortal:CPX-1120),has_output(ComplexPortal:CPX-1062)	UniProtKB:P05067:PRO_0000000093
```

So, I would hazard a couple of guesses. First, I have a hard time believing that col 2 there is legitimate in any way. Second, this is in the goa_uniprot_all file, which we would filter as it should be in goa_human--so wrong file for taxon:9606. By design, we should not see that annotation.

**Comment by pgaudet** [comment]
Thanks @kltm 

@alexsign This is odd - can you check ? When I dump the data as GAF from QuickGO 'P05067:PRO_0000000092' is in Col 17.

**Comment by vanaukenk** [comment]
Hi,
I need to dig a little deeper, but there may be two other things going on here:
1) At least one of the missing ARUK-UCL annotations may have been made after the release from which we downloaded annotations from AmiGO
2) I forgot to also grab any TAS/NAS/IC annotations; at least one of the ARUK-UCL annotations is a TAS annotation.  I'll add those annotations to the spreadsheet tomorrow.

**Comment by alexsign** [comment]
@kltm yes, you are right it's invalid identifier for col 2 in GAF, but it is valid col 2 for GPAD. The GOA is  GPAD centric and every time we unloading GAF we need to convert it appropriately. Thanks for noticing this bug is fixed now and next release the annotations should be corrected in both human specific and goa_uniprot_all gaf files.
@pgaudet The QuickGO is GPAD centric, but it dumps annotation properly in both GAF and GPAD formats. The issue is in release files, see my comments above.

GAF must die!

**Comment by vanaukenk** [comment]
Just to follow up on previous comment wrt other evidence codes, I've now added TAS/NAS/IC/RCA evidenced annotations to the three relevant spreadsheets.

**Comment by pgaudet** [comment]
Great ! Thanks for the quick response. 

@alexsign you didn't reply about the annotation being in the wrong file; I suppose you also took care of that ? 

Thanks, Pascale

**Comment by alexsign** [comment]
@pgaudet no, there was no annotations in the wrong file. The annotations was missing from goa_human_isoform.gaf and had incorrect col 2 in goa_uniprot_all.gaf. The GOA release files on the 3rd of July will have the corrected data. Please let me know if you need them earlier.

**Comment by pgaudet** [comment]
OK, thanks - all good, I don't think we need the data earlier, unless this bug affects many annotations ? 
(Because if you do the release on July 3rd I assume that we get this in  the August GO release, so if a lot of annotations are impacted perhaps we'd like a new file before the next release). 

Thanks, Pascale

**Comment by kltm** [comment]
@pgaudet A release can always be done after the 3rd.

**Comment by pgaudet** [comment]
Hi @alexsign 
This issue doesn't seem to be resolved yet. Can you please have a look ? 

Thanks, Pascale

**Comment by alexsign** [comment]
@pgaudet I don't think I can. The annotations are in our release files (goa_human.[gaf|gpa] and goa_human_isoform.[gaf|gpa]) as well as QuickGO. I don't have an access  to GOC pipeline or AmiGO.

**Comment by pgaudet** [comment]
OK - I had understood that Col2 contained a non-valid ID. (I had assumed this should have been in Col 17 - I guess not ? )

**Comment by alexsign** [comment]
@pgaudet this type of IDs are valid, we had posts about tis before and it clear from file format specs. What column they are depends on the file type. 

goa_human_isoform.gaf it's in col 17
UniProtKB	P05067	APP		GO:0046983	PMID:19660551	IMP		F	Amyloid-beta precursor protein	APP|A4|AD1	protein	taxon:9606	20170710	ARUK-UCL	occurs_in(UBERON:0000955),part_of(GO:0007568),has_output(ComplexPortal:CPX-1120),has_output(ComplexPortal:CPX-1062),has_input(UniProtKB:P05067:PRO_0000000092)	UniProtKB:P05067:PRO_0000000093

goa_human_isoform.gpa it's in col 2
UniProtKB	P05067:PRO_0000000092	enables	GO:0046983	PMID:19660551	ECO:0000315			20170710	ARUK-UCL	occurs_in(UBERON:0000955),part_of(GO:0007568),has_output(ComplexPortal:CPX-1120),has_output(ComplexPortal:CPX-1062),has_input(UniProtKB:P05067:PRO_0000000093)	go_evidence=IMP

**Comment by pgaudet** [comment]
OK, thanks !

**Comment by pgaudet** [comment]
@kltm What do we need to do to add those annotations in our set of annotations? What is not being parsed correctly ?

**Comment by kltm** [comment]
@pgaudet Is this a question of adding a new GAF?

**Comment by pgaudet** [comment]
@kltm The PR you link to is changing the file for SGN. 

I guess we want to make use we include goa_human_isoform.gaf. The seem to be in out products (see http://current.geneontology.org/products/pages/downloads.html) but are not in AmiGO. 

Thanks, Pascale

**Comment by kltm** [comment]
@pgaudet We don't include *any* of the goa\_\*\_isoform files; my recollection was that this was done on purpose.

**Comment by pgaudet** [comment]
@vanaukenk Should we bring that up on an annotation call ? 
I cannot think if a reason not to do that. 

Thanks, Pascale

**Comment by vanaukenk** [comment]
@pgaudet 
Yes, I'll add it to the agenda for Tuesday's call.
